### PR TITLE
Fix issues with custom Ping message windows

### LINKED ISSUE #191: Ping opens too many message windows
When I click the "ping" button, it opens a dialogue to add a message to my ping (good work, BTW @annieli555 , this is going to be really useful!). However, there's a bug which causes "ping" to open a dialogue window for each student in the queue. For example, if there are 4 students in the queue and I press "ping", it opens 4 dialogue windows (I have to x out 4 windows to get back to the normal queue screen). I don't know which student sees the result of which ping dialogue.

### LINKED ISSUE #192: Ping dialog closes unexpectedly
Problem:
when I try to highlight the text in the new ping textbox that opens up, I do so by click on the text and the dragging all the way to the left. If you do this, and release the mouse cursor when it is outside of the ping box region, then it will close *all* of the ping boxes, and does not have the issue described in #191. This was initially confusing to me, and I'm not sure if it's intended to behave this way (I would appreciate if it didn't behave this way).

Reproduce:
Click on the "ping" button for a student, click and hold on the text box as if about to highlight text, then move the mouse cursor anywhere outside of the ping text box (onto the gray area), then release mouse cursor. It will close all related dialogs instead of simply highlighting the text.

### PR DESCRIPTION
**Pull request by pl98**
Closes #191, closes #192 

- Ping now opens only one window when clicked
- Ping window no longer closes when the user clicks outside of the window

### CODE REVIEW

**Comment by mterwill** [comment]
👋 looks like there are some conflicts from the revert and you will need to rebase

**Comment by pl98** [comment]
Fixed merge conflicts!

File: app/assets/javascripts/components/request.js.jsx [hunk 1]
```diff
@@ -39,7 +39,10 @@ var Request = React.createClass({
         <Actions>
           <Action data={{ title: 'Resolve', action: this.props.resolve.bind(null, this.props.request.id)}} />
           <Action data={{ title: pinText, action: this.props.pin.bind(null, this.props.request.id)}} />
-          <Action data={{ title: 'Ping', action: this.props.bump.bind(null, this.props.request.id)}} />
+          <Ping 
+            data={{ title: 'Ping', action: (message) => this.props.bump(this.props.request.id, message)}}
+            id={moment.utc(this.props.request.created_at).valueOf()}
```
Comment (mterwill) [inline app/assets/javascripts/components/request.js.jsx:44]: Why not use `this.props.request.id` for a unique ID here?

Comment (pl98) [reply app/assets/javascripts/components/request.js.jsx]: Oops, forgot about that somehow. Fixed!

**Review by mterwill — APPROVED** [review]
